### ISSUE TITLE
[Bug] - permissions: Age 1, joining server, forced to place Coralium Infused Stone

### ISSUE THREAD

**Issue by JustHeyo** [Darkosto/SevTech-Ages#1877]
3.0.6 (latest); server;  40% of 7808MB of RAM used.  Java 1.8.0_161 64bit; no resource, no optifine, MultiMC 5 used to launch.  I'm on Windows 7.

! I'm worried that this could reflect a larger issue - are player progression-related permissions loaded at a later point than they ought to be when re-joining a server?

DETAILS:
I had made a Coralium Infused Stone in order to make the Beneath Portal while in Age 1.  The last progression achievement I have unlocked was creating the Necronomicon.  I had the Coralium Infused Stone on my hotbar and selected when I logged out.

Without closing Minecraft, I later rejoined the server.  Upon doing so, I saw a message "You have dropped the unfamiliar item" (or similar verbiage) and the Coralium Infused Stone disappeared from my hotbar; the world finished loading in and the Coralium Infused Stone was placed in the world in front of me (as a block, not an item) and there was no message in the chat log.

I attempted to mine the block, but it broke and was destroyed.

**Comment by LycanBlackpaw** [comment]
So, this is more a latency bug than anything-basically here's what happens. When you first join a world (server or singleplayer, doesn't matter), the game takes a second or two to 'catch up' to your actual progress. During this time, it pretends you're in the Tutorial age. The item didn't actually drop, and was still in the same inventory slot, just invisible. You then likely right clicked, thinking your hand was empty, placing the block. When you relogged, the world re-synced and showed the placed block.

To fix this, you simply open your inventory and click one of the slots of the 'missing' items. Armor does this often as well, so I usually click an armor slot. This re-syncs your inventory to it's correct state. Opening chests and other inventories does the same for your normal inventory, but not armor slots.

**Comment by artdude543** [comment]
Yeah, this is just an effect of the syncing sadly; sometimes your lucky and it loads fast other times can be slow. So I suggest waiting a few seconds before playing after load to ensure all the scripts have loaded and your player has synced with the world.

**Comment by JustHeyo** [comment]
For clarity, I'm not hitting anything when this happens - I believe the block was placed all on its own.  Today I took screenshots of an "unfamiliar item" message and empty hotbar space, and after a few minutes noticed that my crafting table on-a-stick was back in the affected slot.  I see this message nearly every time I either join the game or change dimensions.  Further, I've noticed some very peculiar behavior when going to/from The Betweenlands - my blood-slime boots disappear and I do not bounce when falling, unless I click on the boots slot - at which point they appear on my cursor and I can place them into the slot again.
